docs/source/content/overview/linear-regression.rst

```diff
@@ -6,81 +6,83 @@ Linear Regression
   :local:
   :depth: 3
 
+*******
+Summary
+*******
+Linear regression is a technique used to analyze a linear relationship between input variables and a single output variable. We measure how accurate the resulting model is using a cost function. We want to minimize the value of the cost function in order to get a more accurate model.
+
 ********
 Overview
 ********
-**Linear regression** is a technique used to analyze a **linear relationship** between **input/independent/x** variables and a single **output/dependent/y** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
-
-.. image:: _img/LR.png
+**Linear regression** is a technique used to analyze a **linear relationship** between **input** variables and a single **output** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
```

Comment:
Since we already have this sentence in the above paragraph, we should use another opener. Something like 

```
For any data set, we can find a linear relationship between all the points. A linear relationship means ... 
There are several different types of linear regression - one such type is simple linear regression. Simple linear ... 
Let's take a look at an example linear relationship with the following data points:
[Plot]
```

Generally this is what I mean by transitions - it's better to move into a fact/image before stating/displaying it outright